teams.py

```diff
@@ -0,0 +1,27 @@
+import timer
+
+
+class Team:
+    def __init__(self, is_white_team):
+        self.pieces = []
+        self.timer = timer.Timer()
+        self.score = 0
+        self.is_white_team = is_white_team
+
+    def update_score(self):
+        self.score = sum(piece.score for piece in self.pieces if not piece.is_eaten)
+
+    def __str__(self):
+        return 'white team' if self.is_white_team else 'black team'
+
+    def print_pieces(self):
+        for piece in self.pieces:
+            print(f'\n{piece}')
+
+
+def get_score_dif(white_team: Team, black_team: Team):
```

Comment:
Rename to "get_score_difference", it's not a very long name and it's better to have full words instead of shortening them. To you it's very clear that "dif" is short of "difference", between other person who works on this project may think you meant "difficult" or something else that makes more sense.

Reply:
:ice_cream: